### DISCUSSION TITLE
Why I can't type more than 400 char in text area?

### DISCUSSION THREAD

**Discussion by Abod-Shashtari** [charmbracelet/bubbletea#441] [Q&A] [answered]
I can't type in text area more than 400 char but I can use `m.textArea.SetValue(string(content))` to insert more than 400 char, so my question is : how to make the user able to type more than 400 char in text area?

**Comment by maaslalani** [comment, marked as answer]
The `CharLimit` is 400 by default, you can change it by setting the `CharLimit`

```
t := textarea.New()
t.CharLimit = 1000
```

**Reply by Abod-Shashtari** [reply]
Thank you :)
